compass-js-frontend/src/main/webapp/ui/view/editor/hierarchy.js

```diff
@@ -52,10 +52,32 @@ var HierarchyView = AmpersandView.extend({
 		});
 		// what we actually want to save is the internal tree object.
 		this.tree = $tree.fancytree('getTree');
+		$tree.on('contextmenu', this.showContextMenu.bind(this));
+	},
+	registerContextOnMenu: function() {
+		var $menu = $('.basicContextContainer');
```

Comment:
After toying with the code myself, this does not appear to be the case. That's what I get for wanting the "simple" Context Menu solution. :grinning:
If I wanted to be pedantic, it could be more correct if we would proxy the right-click on the menu container overlay back to whatever lies below. (Causing, e.g., the menu to re-open when clicking through to another node in the hierarchy.) However, this is close enough.